### PR TITLE
Release 2.0.0

### LINKED ISSUE #4: Regexpr for yw/yq/ym index in `iNZightTS()` needs to be updated
Optional space between year and week/month/quarter.

### PR DESCRIPTION
**Pull request by tmelliott**

### CODE REVIEW

**Comment by tmelliott** [comment]
Hi @szmsu2011 
Instead of trying to figure out how to host multiple versions of the same package, I've taken the simpler (but more bloated) approach of creating a new package and hooking up a new, separate interface with it. Could you please take a look and see if there are any last changes you need to make before we finalise 'version 2'?

File: tests/testthat/test_plots.R [hunk 1]
```diff
@@ -3,95 +3,173 @@ context("Basic time series graphs")
...
+    expect_is(plot(y, emphasise = 2, non_emph_opacity = 0), "ggplot")
+    x <- visitorsM2 %>%
+        dplyr::mutate(y = ggplot2::cut_number(Japan, 4)) %>%
+        inzightts()
+    y <- visitorsA2 %>%
+        tidyr::pivot_longer(!Time, names_to = "Country", values_to = "Visitors") %>%
+        dplyr::mutate(y = ggplot2::cut_number(Visitors, 4)) %>%
+        inzightts(key = "Country")
+    expect_is(plot(x, "y"), "ggplot")
+    # TODO: spitting out a warning
```
Comment (szmsu2011) [inline tests/testthat/test_plots.R]: All the warnings below this line were expected and also tested via other `expect_warning()` calls. Some previous versions of `testthat` allowed warnings to exist when testing with `expect_is()` thus I did not suppress them.

File: tests/testthat/test_plots.R [hunk 2]
```diff
@@ -3,95 +3,173 @@ context("Basic time series graphs")
...
 
+test_that("Check season plot configuration", {
+    expect_is(seasonplot(t, labels = "none"), "patchwork")
+    expect_error(seasonplot(t, model_range = 2001))
+    # TODO: does not produce a warning
+    expect_warning(seasonplot(t, model_range = lubridate::ymd(c(20050101, 20101231))))
+    expect_is(seasonplot(tm, names(tm)[-1]), "patchwork")
+    t_key_gap <- y
+    t_key_gap$Australia[3] <- NA
+    # TODO: gives ERROR instead of WARNING
```
Comment (szmsu2011) [inline tests/testthat/test_plots.R]: Not sure why test these tests were successful for me without returning an error. To be discussed.

**Comment by szmsu2011** [comment]
Could you please double check my push permission? I'm getting the following error messages:

```
remote: Permission to iNZightVIT/iNZightTS2.git denied to szmsu2011.
fatal: unable to access 'https://github.com/iNZightVIT/iNZightTS2.git/': The requested URL returned error: 403
```

**Comment by tmelliott** [comment]
Try again now @szmsu2011

**Comment by tmelliott** [comment]
Closing the PR, will deal with #3 first, then do a release.

**Pull request closed without merging**